tools/linter/clang_tidy/generate_build_files.py

```diff
@@ -22,10 +22,6 @@ def run_cmd(cmd: List[str]) -> None:
         exit(1)
 
 
-def run_timed_cmd(cmd: List[str]) -> None:
-    run_cmd(["time"] + cmd)
```

Comment:
`time` is not an executable, so the subprocess has trouble running it https://askubuntu.com/questions/801493/python-subprocess-call-not-working-as-expected.  We don't need this here though.  I don't think anyone is looking at that info